docs/narr/traversal.rst

```diff
@@ -237,7 +237,7 @@ uses this algorithm to find a :term:`context` resource and a :term:`view name`.
 
     The traversal algorithm by default attempts to first URL-unquote and then
     Unicode-decode each path segment derived from ``PATH_INFO`` from its
-    natural byte string (``str`` type) representation.  URL unquoting is
+    natural string representation.  URL unquoting is
```

Comment:
Unicode is mentioned twice in this paragraph. I think they belong, because we're discussing en-/de-coding of UTF-8, not Unicode objects, but I'm not sure. Please let me know.

Reply:
yeah leave it the way it is